errors/Makefile.am

```diff
@@ -24,6 +24,12 @@ EXTRA_DIST = \
 	template.am \
 	aliases alias-link.sh alias-upgrade errorpage.css TRANSLATORS COPYRIGHT
 
+if ENABLE_TRANSLATION
+ALIAS_LINKER=$(SHELL) $(srcdir)/alias-link.sh "$(LN)" "$(RM)" "$(DESTDIR)$(DEFAULT_ERROR_DIR)"
+else !ENABLE_TRANSLATION
+ALIAS_LINKER=true
+endif
+
```

Comment:
> PR description: This change also stops exposing --disable-translation builds to alias-link.sh bugs that generate warnings for boilerplate statements.

```
WARNING: ## translations do not exist. Nothing to do for: Copyright (C) 1996-2022 The Squid Software Foundation and contributors
WARNING: ## translations do not exist. Nothing to do for: 
WARNING: ## translations do not exist. Nothing to do for: Squid software is distributed under GPLv2+ license and includes
WARNING: ## translations do not exist. Nothing to do for: contributions from numerous individuals and organizations.
WARNING: ## translations do not exist. Nothing to do for: Please see the COPYING and CONTRIBUTORS files for details.
WARNING: ## translations do not exist. Nothing to do for: 
```

The above warnings are displayed by "make install" in --enable-translation builds. I do not plan on fixing this out-of-scope problem.

Reply:
I think this is a newly added output. If so then it does need to be fixed before this PR is suitable quality for merge.

All lines that begins with a ```#``` character should be ignored by the linker script and so not generate any output.